packages/astro/src/build.ts

```diff
@@ -283,21 +283,30 @@ export function findDeps(html: string, { astroConfig, srcPath }: { astroConfig:
     }
   });
 
-  $('link[href]').each((i, el) => {
+  $('link[href]').each((_i, el) => {
     const href = $(el).attr('href');
     if (href && !isRemote(href) && ($(el).attr('rel') === 'stylesheet' || $(el).attr('type') === 'text/css' || href.endsWith('.css'))) {
       const dist = getDistPath(href, { astroConfig, srcPath });
       pageDeps.css.add(dist);
     }
   });
 
-  $('img[src]').each((i, el) => {
+  $('img[src]').each((_i, el) => {
     const src = $(el).attr('src');
     if (src && !isRemote(src)) {
       pageDeps.images.add(getDistPath(src, { astroConfig, srcPath }));
     }
   });
 
+  $('img[srcset]').each((_i, el) => {
+    const srcset = $(el).attr('srcset') || '';
+    const sources = srcset.split(',');
+    const srces = sources.map(s => s.trim().split(' ')[0]);
```

Comment:
are we confident that this will work in all cases? From a quick google it looks like the regex equivalent is a bit more complicated than what you have here: https://stackoverflow.com/questions/26466724/how-do-i-extract-an-imageset-from-srcset-attribute/26467184

Reply:
Yep, very confident having read the MDN page which describes this attribute. This code is simpler because it only worries about grabbing the URLs.